pex/bin/pex.py

```diff
@@ -720,6 +662,10 @@ def main(args=None):
   if options.python and options.interpreter_constraint:
     die('The "--python" and "--interpreter-constraint" options cannot be used together.')
 
+  # We already adjusted the sys.path at the top of this file, but potentially re-adjust path to
+  # include wheel.
+  vendor.adjust_sys_path(include_wheel=options.use_wheel)
```

Comment:
Yes - it will with different values for include_wheel. It's probably not a big concern since it only happens at build time, but `vendor` and `third_party` could stand some tests and docs so I'll likely fix in my review feedback pass here.